vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,170 @@ knitr::opts_chunk$set(
+
+Another aspect about the intuition behind the more ambitious scenarios is that a higher ambition in transitioning to a low-carbon economy (scaling up low-carbon technologies and scaling down high-carbon technologies) correlates with a lower expected global heating outcome.
+
+```{r, message=FALSE, warning=FALSE, echo=FALSE}
+library(pacta.loanbook)
+
+market_share_targets_portfolio <-
+  target_market_share(
+    data = prioritize(match_name(loanbook_demo, abcd_demo)),
+    abcd = abcd_demo,
+    scenario = scenario_demo_2020,
+    region_isos = region_isos_demo
+  )
+
+qplot_trajectory(
+  dplyr::filter(
+    market_share_targets_portfolio,
+    sector == "power",
+    technology == "renewablescap",
+    region == "global",
+    scenario_source == "demo_2020"
+  )
+)
+```
+
+## Sectoral Decarbonization (SDA) Target
+
+The sectoral decarbonization approach (SDA) is used to calculate PACTA alignment metrics for the aviation, cement and steel sectors. The commonality of these sectors is that there are no market ready production-pathways at the technology level (or fuel type level) available yet that can be used to describe how to decarbonize the sector activities. They are currently considered "hard-to-abate" sectors. Since there are no market-ready low-carbon alternatives available in these sectors at the moment, that can be scaled up fast, the SDA approach is used to calculate sector-level metrics instead of the technology level metrics available for sectors that use the market share approach. These sector level metrics are production-based emission intensities, such as tonnes of CO^2^ per tonne of cement or tonnes of CO^2^ per tonne of steel. Regardless of how the output unit is produced, this is a comparable measure for the entire sector. The principle idea of the SDA approach is that the required carbon intensity that a sector needs to reach in the future, given assumptions around future production volumes, can be derived by dividing a sector specific carbon budget by the expected production volume. The current GHG emissions divided by the current aggregate sector production volume gives the starting point of the emission intensity. If the actors in a sector decarbonize the production in a way that follows the average emission intensity of the sector, then the scenario assumptions will be solved for in aggregate. At the portfolio/company level, the SDA approach requires every actor to decarbonize their production following their individual trajectory that starts at their current emission intensity and converges on a common emission intensity that is the same for all actors in a sector at the end of the scenario time frame (for example in 2050). This means that a company with an already lower emission intensity will not have to reduce their emission intensity as much as a company that lags the current sector average.
```

Comment:
@jdhoffa here it mentions cement and steel *and* aviation. I was going to parameterize this like I did in the emission intensity plot section so it pulls from `co2_intensity_scenario_demo`, e.g. `r cli::format_inline("{unique(co2_intensity_scenario_demo$sector)} sector{?s}")`, but that doesn't include aviation?

Reply:
the co2 scenario demo data set does not include aviation. most of the main scenarios we use do not cover all SDA sectors at the same time. At least in the past, that was a recurring phenomenon. Must have inspired us to to the same in the demo data

Reply:
so... should we not mention aviation here then?

Reply:
We should mention it. Depending on the scenario that users compare against, they will be able to use it, so it makes sense to mention it here